static/js/apps/eval_retrieval_generation/util.ts

```diff
@@ -78,6 +91,20 @@ export const processText = (text: string): string => {
   );
 };
 
+function getTooltipHtml(
+  dcStat: string,
+  callId: string,
+  calls?: DcCalls
+): string {
+  const dcResponse: string = calls ? calls[callId]?.dcResponse ?? "" : "";
```

Comment:
nit: instead of trying to write all of this in one line with lots of ? and ??, probably more readable with: 

```
let dcResponse: string = "";
if (calls) {
    dcResponse = calls[callId]?.dcResponse || "";
}
```

Reply:
done!